hatchways/team-eagle
# Auth Routes Tests

In this pull request, I:
- Wrote tests for `/auth/*` routes (to avoid manually testing them with each branch merge)


#### How to test
1. run `yarn install`
2. run `yarn test`

Everything should pass

Please let me know if you get any error or have any questions
Thanks,
H
